database/exip.sql

```diff
@@ -337,6 +337,8 @@ CREATE TABLE IF NOT EXISTS `PolicyAndExport` (
   `policyType` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `application` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
 	`requestedStartDate` datetime(3) DEFAULT NULL,
+	`totalValueOfContract` int DEFAULT NULL,
```

Comment:
As I believe we do not expect negative values in this field?

Reply:
That's correct, no negative values. We can add this to the SQL dump, but it's currently not possible to add `UNSIGNED` flag with keystone schema, so it will not get added to the DB if this dump is not used and it would get overridden when the schema changes.

Happy to add this in, but I don't see much point and think it should be the same as what keystone generates.

The UI will not call the API to save `totalValueOfContract` if there are validation errors e.g if there is a negative value